### ISSUE TITLE
"pytest" is unbound

### ISSUE THREAD

**Issue by WhyNotHugo** [microsoft/pyright#729]
**Describe the bug**

I'm getting a `"pytest" is unbound` error. A _method_ can be bound or unbound, but for modules this isn't the case, so the error does not make sense.

**To Reproduce**

Simple (almost textbook) example:

```python
import pytest


def test_zero_division():
    with pytest.raises(ZeroDivisionError):
        1 / 0
```

**Expected behavior**

This shouldn't raise. I came across dozens of instances of this error in my code-base, but the above is literally a copy-paste [from the pytest documentation](https://docs.pytest.org/en/stable/assert.html#assertions-about-expected-exceptions).

**VS Code extension or command-line**

pyright-server 1.1.41

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next published version of Pyright.

**Comment by WhyNotHugo** [comment]
Wow, that was fast! :sparkles: 
Thanks!

**Comment by erictraut** [comment]
This is fixed in Pyright 1.1.43, which I just published.

**Comment by WhyNotHugo** [comment]
Thanks!
